<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DirecteurRH</class>
 <widget class="QMainWindow" name="DirecteurRH">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 183, 124);
border-color:rgb(255, 183, 124);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>791</width>
      <height>491</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 183, 124);
border-color: rgb(255, 255, 255);</string>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>ACCUEIL</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>100</y>
        <width>391</width>
        <height>101</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>28</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 183, 124);
</string>
      </property>
      <property name="text">
       <string>Ressources Humaines </string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Utilisateurs Enregistrer</string>
     </attribute>
     <widget class="QTableView" name="tableViewUsers">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>761</width>
        <height>281</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustIgnored</enum>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="showGrid">
       <bool>true</bool>
      </property>
      <property name="gridStyle">
       <enum>Qt::CustomDashLine</enum>
      </property>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>82</number>
      </attribute>
     </widget>
     <widget class="QPushButton" name="pushButtonExtract">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>380</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Extraire </string>
      </property>
      <property name="icon">
       <iconset resource="../images/images.qrc">
        <normaloff>:/depositphotos_289451976-stock-illustration-white-upload-inbox-icon-isolated.jpg</normaloff>:/depositphotos_289451976-stock-illustration-white-upload-inbox-icon-isolated.jpg</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>10</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Myanmar Text</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Dashboard Users</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonPrint">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>380</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Imprimer</string>
      </property>
      <property name="icon">
       <iconset resource="../images/images.qrc">
        <normaloff>:/118-1186893_computer-printer-png-clipart-printer-icon.png</normaloff>:/118-1186893_computer-printer-png-clipart-printer-icon.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Demande de Congé Effectués</string>
     </attribute>
     <widget class="QTableView" name="tableViewUsers_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>761</width>
        <height>281</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustIgnored</enum>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="showGrid">
       <bool>true</bool>
      </property>
      <property name="gridStyle">
       <enum>Qt::CustomDashLine</enum>
      </property>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>82</number>
      </attribute>
     </widget>
     <widget class="QPushButton" name="pushButtonExtract_2">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>380</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Extraire </string>
      </property>
      <property name="icon">
       <iconset resource="../images/images.qrc">
        <normaloff>:/depositphotos_289451976-stock-illustration-white-upload-inbox-icon-isolated.jpg</normaloff>:/depositphotos_289451976-stock-illustration-white-upload-inbox-icon-isolated.jpg</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>10</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Myanmar Text</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Dashboard Congés</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonPrint_2">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>380</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Imprimer</string>
      </property>
      <property name="icon">
       <iconset resource="../images/images.qrc">
        <normaloff>:/118-1186893_computer-printer-png-clipart-printer-icon.png</normaloff>:/118-1186893_computer-printer-png-clipart-printer-icon.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDRH">
    <property name="title">
     <string>DRH</string>
    </property>
   </widget>
   <addaction name="menuDRH"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../images/images.qrc"/>
  <include location="../images/images.qrc"/>
 </resources>
 <connections/>
</ui>
